docs/beast_priors.rst

```diff
@@ -32,51 +32,50 @@ The age prior is the star formation rate (SFR) and can be
 
   age_prior_model = {'name': 'flat'}
 
-or
+  or to set the star formation rate in M_sun/year, use
 
-.. code-block:: python
-
-  age_prior_model = {'name': 'flat_linear'}
+  age_prior_model = {'name': 'flat'},
+                     'amp': sfr}
 
 2. Flat in log age
 
 .. code-block:: python
 
   age_prior_model = {'name': 'flat_log'}
 
-3. Set by bins spaced in logage.
+3. Set by bins spaced in logage (log10(years)).
 
 For example, step like priors can be specified by:
 
 .. code-block:: python
 
   age_prior_model = {'name': 'bins_histo',
-                     'logages': [6.0, 7.0, 8.0, 9.0, 10.0],
+                     'x': [6.0, 7.0, 8.0, 9.0, 10.0],
                      'values': [1.0, 2.0, 1.0, 5.0, 3.0]}
 
 Or using bin edges (where N = N_values+1) like those output by `np.histogram()`:
 
 .. code-block:: python
 
   age_prior_model = {'name': 'bins_histo',
-                     'logages': [6.0, 7.0, 8.0, 9.0, 10.0],
+                     'x': [6.0, 7.0, 8.0, 9.0, 10.0],
                      'values': [1.0, 2.0, 1.0, 5.0]}
 
 For example, lines connecting the bin value of the priors can be specified by:
 
 .. code-block:: python
 
  age_prior_model = {'name': 'bins_interp',
-                    'logages': [6.0, 7.0, 8.0, 9.0, 10.0],
+                    'x': [6.0, 7.0, 8.0, 9.0, 10.0],
                     'values': [1.0, 2.0, 1.0, 5.0, 3.0]}
 
 4. An exponentially decreasing SFR (in time, but here increasing with age)
 with a 0.1 Gyr time constant (with `tau` parameter in Gyr):
 
 .. code-block:: python
 
-  age_prior_model = {'name': 'exp',
-                     'tau': 0.1}
+  age_prior_model = {'name': 'exponential',
+                     'a': 0.1}
```

Comment:
Why "a"? The function will look for a keyword "tau", right?

Reply:
Good point.  I think I was thinking to make this more generic as `a`, but then never updated the code.  And `tau` is just as generic now that you have asked this great question.  Making it all `tau`.